pkg/operators/v1alpha1/subscription_types.go

```diff
@@ -118,8 +118,18 @@ const (
 	// SubscriptionBundleUnpackFailed indicates that the unpack job failed
 	SubscriptionBundleUnpackFailed SubscriptionConditionType = "BundleUnpackFailed"
 
-	// SubscriptionOperatorDeprecated indicates that the Operator currently installed with this Subscription has been deprecated.
-	SubscriptionOperatorDeprecated SubscriptionConditionType = "OperatorDeprecated"
+	// SubscriptionDeprecated is a roll-up condition which indicates that the Operator currently installed with this Subscription
+	//has been deprecated. It will be present when any of the three deprecation types (Package, Channel, Bundle) are present.
+	SubscriptionDeprecated SubscriptionConditionType = "Deprecated"
+
+	// SubscriptionOperatorDeprecated indicates that the Package currently installed with this Subscription has been deprecated.
+	SubscriptionPackageDeprecated SubscriptionConditionType = "PackageDeprecated"
+
+	// SubscriptionOperatorDeprecated indicates that the Channel currently installed with this Subscription has been deprecated.
```

Comment:
Nit: Channels aren't installable IIUC. Not serious enough to block this IMO.
```suggestion
	// SubscriptionOperatorDeprecated indicates that the Channel used with this Subscription has been deprecated.
```

Reply:
Woops, I'll correct this in a follow up. Thanks for catching.